lib/hnvm/config.sh

```diff
@@ -14,11 +14,21 @@ script_dir="$( cd -P "$( dirname "$source" )" >/dev/null 2>&1 && pwd )"
 
 export COMMAND_OUTPUT=/dev/stdout
 
+# Set these defaults here instead of the rc file so that HNVM_NOFALLBACK never blocks these defaults
+export HNVM_PATH=${HNVM_PATH:-$HOME/.hnvm}
```

Comment:
These felt like they belonged in `config` and I couldn't see any immediate reason why they shouldn't be. Happy to find a workaround instead and comment with details on why it should live in enable_bin :)